docs/user-manual/en/versions.md

```diff
@@ -8,6 +8,22 @@ This chapter provides the following information for each release:
   - **Note:** Follow the general upgrade procedure outlined in the [Upgrading the Broker](upgrading.md) 
     chapter in addition to any version-specific upgrade instructions outlined here.
 
+
+## Upgrading tool
+
+Since version 2.27.0, it is possible to uptadate all the paths of an instance toward a newer version.
+
+To upgrade to a new version, simply do the following code:
+
+```shell
+cd $NEW_ARTEMIS_DOWNLOAD/bin/
+./artemis upgrade PATH_TO_UPGRADING INSTANCE
+```
+
+artemis, artemis.profile will be updated to the new version.
+
+(Notice: to avoid removing eventual user's  customizations of the scripts, we don't automatically bring new additions to the script. Please check your artemis and artemis.profile accordingly after the upgrade)
```

Comment:
Extra space after user's

Reply:
I think this documentation would make sense in `upgrading.md` as that's where the general upgrade instructions are. I also think it would be good to explain why the script should be used instead of the normal manual process of just changing the `ARTEMIS_HOME` location in `artemis.profile`.

Reply:
I have already placed it in upgrading.. and I will add wording about why use upgrading.